content/community/engineering_guidelines.md

```diff
@@ -30,10 +30,6 @@ As an alternative to or extension of the [guideline above]({{< relref "#discuss-
 
 Features normally span more than one component. Having relevant people assigned to an issue from the start will hopefully gain more complete ownership from beginning to end. As an example, for a feature that adds new pages to the UI, it is expected that UX, UI, backend, QE, and docs would all be involved. Let's call these "feature miniteams". Miniteams are adhoc teams and people can be expected to be a member of multiple miniteams at one time.
 
-## Commit messages should follow the Conventional Commits standardized commit message format
```

Comment:
don't we like conventional commits?

Reply:
Ah I thought we did away with this? At least the UI team did. I'll add this back in. 👍